pandas/tests/groupby/test_apply.py

```diff
@@ -1317,3 +1317,15 @@ def test_apply_str_with_args(df, args, kwargs):
     result = gb.apply("sum", *args, **kwargs)
     expected = gb.sum(numeric_only=True)
     tm.assert_frame_equal(result, expected)
+
+
+def test_result_name_when_one_group():
```

Comment:
can you parameterize this tests instead of looping

Reply:
done